scripts/WalletBalance.vue

```diff
@@ -0,0 +1,242 @@
+<script setup>
+import { cChainParams, COIN } from './chain_params.js';
+import { translation } from './i18n';
+import { ref, computed } from 'vue';
+
+import { nDisplayDecimals } from './settings';
+import { beautifyNumber } from './misc';
+import { getEventEmitter } from './event_bus';
+import { cMarket, strCurrency } from './settings.js';
+import {
+    mempool,
+    optimiseCurrencyLocale,
+    refreshChainData,
+    openExplorer,
+    toggleExportUI,
+    toggleBottomMenu,
+} from './global';
+import { renderWalletBreakdown } from './charting';
+import {
+    guiRenderCurrentReceiveModal,
+    guiRenderContacts,
+} from './contacts-book';
+import { wallet, getNewAddress } from './wallet';
+
+const balance = ref(0);
+const price = ref(0.0);
+const updating = ref(false);
+const currency = ref('USD');
+const balanceStr = computed(() => {
```

Comment:
This value is never updated

Reply:
balanceStr should updated through Vue reactivity

Reply:
Fixed, nDisplayDecimals was not reactive (Also had a typo on settings.js)